src/lib/components/CommunitySelectionModal/CommunityListItem.js

```diff
@@ -12,70 +12,57 @@ import PropTypes from 'prop-types';
 import { i18next } from '@translations/i18next';
 import { CommunityContext } from './CommunityContext';
 
-export const CommunityListItem = ({ result, standAlone }) => {
+export const CommunityListItem = ({ result }) => {
   const { setLocalCommunity, getChosenCommunity } =
     useContext(CommunityContext);
 
   const { metadata } = result;
   const linkToCommunityPage = result.links.self_html;
   const linkToLogo = result.links.logo;
-
   const itemSelected = getChosenCommunity()?.uuid === result.uuid;
 
   return (
-    <Item key={result.id} className="pr-20">
+    <Item key={result.id} className={itemSelected ? 'selected' : ''}>
       <Item.Image
-        as={Image}
         size="tiny"
-        src={linkToLogo}
-        fallbackSrc="/static/images/square-placeholder.png"
+        src={linkToLogo || "/static/images/square-placeholder.png"}
       />
 
-      <Item.Content verticalAlign="top">
-        <Item.Header
-          as="a"
-          href={linkToCommunityPage}
-          target="_blank"
-          rel="noreferrer"
-        >
+      <Item.Content>
+        <Item.Header>
           {metadata.title}
+          <Button
+            as="a"
+            href={linkToCommunityPage}
+            target="_blank"
+            rel="noreferrer"
+            size="small"
+            className="transparent pt-0 ml-15 mb-5"
+            content={i18next.t("View community")}
+            icon="external alternate"
+            title={i18next.t("Opens in new tab")}
+          />
         </Item.Header>
-        <Item.Description>
+        <Item.Description as="p">
```

Comment:
this will most likely remove the SUI styling for Item.Description, to be verified

Reply:
The SUI styling is still applied :)